
# HG changeset patch
# User John Lindgren <john.lindgren@tds.net>
# Date 1248286313 14400
# Node ID d7e5d6e22a135f196becd237a28fca2469482a47
# Parent  6cf3885ad1009feea34c5b67f3bd7f4c3fbdd82c
Remove references to the mysterious, unused libudet, since Audacious seems to build fine without it.

--- a/configure.ac	Wed Jul 22 11:30:19 2009 -0400
+++ b/configure.ac	Wed Jul 22 14:11:53 2009 -0400
@@ -94,11 +94,6 @@
 [charset detection support],
 [
     AC_DEFINE([USE_CHARDET], 1, [Define if character set detection enabled])
-    AC_CHECK_LIB([udet_c], [detectCharset], [
-        AC_DEFINE([HAVE_UDET], 1, [Define if the system has Mozilla universal character detector library])
-        CHARDET_LIBS="$CHARDET_LIBS -ludet -ludet_c"
-        AC_SUBST([CHARDET_LIBS])
-    ])
 ])
 AUD_CONDITIONAL([USE_CHARDET], [enable_chardet])
 
--- a/src/audacious/chardet.c	Wed Jul 22 11:30:19 2009 -0400
+++ b/src/audacious/chardet.c	Wed Jul 22 14:11:53 2009 -0400
@@ -24,9 +24,6 @@
 
 #ifdef USE_CHARDET
 #include "../libguess/libguess.h"
-#  ifdef HAVE_UDET
-#    include <libudet_c.h>
-#  endif
 #endif
 
 gchar *
@@ -66,11 +63,9 @@
     if (dfa_validate_utf8(str, strlen(str)))
         return g_strdup(str);
 
-#  ifdef HAVE_UDET
     /* chardet encoding detector */
     if ((out_str = chardet_to_utf8(str, strlen(str), NULL, NULL, NULL)))
         return out_str;
-#  endif
 #endif
 
     /* assume encoding associated with locale */
@@ -145,7 +140,7 @@
 			ret = NULL;
 		}
 	}
-	
+
 	return NULL;	/* if I have no idea, return NULL. */
 }
 
--- a/src/audacious/ui_preferences.c	Wed Jul 22 11:30:19 2009 -0400
+++ b/src/audacious/ui_preferences.c	Wed Jul 22 14:11:53 2009 -0400
@@ -162,9 +162,7 @@
     { N_("Hebrew")   , N_("Hebrew") },
     { N_("Turkish")  , N_("Turkish") },
     { N_("Arabic")   , N_("Arabic") },
-#ifdef HAVE_UDET
     { N_("Universal"), N_("Universal") }
-#endif
 };
 
 static ComboBoxElements bitdepth_elements[] = {
--- a/src/audacious/ui_urlopener.c	Wed Jul 22 11:30:19 2009 -0400
+++ b/src/audacious/ui_urlopener.c	Wed Jul 22 14:11:53 2009 -0400
@@ -48,9 +48,6 @@
 
 #ifdef USE_CHARDET
 #include "../libguess/libguess.h"
-#  ifdef HAVE_UDET
-#    include <libudet_c.h>
-#  endif
 #endif
 
 #define URL_HISTORY_MAX_SIZE 30

